### PR TITLE
InteractiveApiProvider supports saving as attachment

### PR DESCRIPTION
**Pull request by kswenson**

### CODE REVIEW

**Review by scytacki — APPROVED** [review]
Looks good to me.

**Review by scytacki — APPROVED** [review]
Looks good to me.

I haven't been able to get it work with the deployed branch though. But perhaps it is a caching issue.

File: package.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@concord-consortium/lara-interactive-api": "^1.1.0",
-    "@concord-consortium/token-service": "^1.3.0",
-    "aws-sdk": "^2.948.0",
+    "@concord-consortium/lara-interactive-api": "^1.1.1",
+    "@concord-consortium/token-service": "^2.0.0",
+    "aws-sdk": "^2.958.0",
```
[the remark this answers is not in the document]
Comment (kswenson) [reply package.json:14]: I thought that initially, but it's a dependency of the preexisting `s3-share-provider`.

File: package.json [hunk 2]
```diff
@@ -9,9 +9,9 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@concord-consortium/lara-interactive-api": "^1.1.0",
-    "@concord-consortium/token-service": "^1.3.0",
-    "aws-sdk": "^2.948.0",
+    "@concord-consortium/lara-interactive-api": "^1.1.1",
```
[the remark this answers is not in the document]
Comment (kswenson) [reply package.json:12]: Once the new version of the library has been published.

**Comment by scytacki** [comment]
I got it to work. I had the wrong version of the activity player. For reference here is a URL for testing this:
https://activity-player.concord.org/branch/attachments/?activity=https%3A%2F%2Fauthoring.staging.concord.org%2Fapi%2Fv1%2Factivities%2F21087.json&page=page_309199

A few things I noticed while testing the example app:
- sometimes changes are not saved even though it says they are.
- the long delay while loading the initial attachment content is confusing

Perhaps the second one is something that a real CFM app like CODAP handles better. 
The slowness (long delay) seems to be due to the token-service being very slow at the initial record request. I'd bet we can speed that up with a little attention to it. 

The first issue seems like race condition, but can be handled later.

**Comment by kswenson** [comment]
@scytacki I forgot that we had talked about changing the configuration so that clients like CODAP and Sage wouldn't have to rebuild to enable the new AP/InteractiveAPI sharing features. I added one commit so that now enabling the LaraProvider also enables the InteractiveApiProvider and no additional configuration is required.

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (kswenson) [reply package.json]: Done now that the new version of the library has been published.

**Review by scytacki — APPROVED** [review]
looks good to me

**Pull request merged**